### ISSUE TITLE
Inconsistencies Between Reports 

### ISSUE THREAD

**Issue by thomasnatalie** [SMSK-CO/SMSK#26]
We are confused by what we are seeing in the reports. I have attached a spreadsheet with an example of a student. This student (4365) is showing up three times when we pull a child individual report with different data for the referral. When I look at the large data export (second tab) of all data from SMSK, this child shows up three times but with data entered for decay and sealants. Also, if the data was just pulled and the kid was seen on different days or if the days were entered incorrectly, it doesn't explain the referrals or why the sealants and decay do not show. I am seeing duplicate Child ID's when I pull a child individual report - not sure why this is happening - so, I'm not sure how widespread this is. Please let me know if you would like more clarification.
[ExampleForGitHub.xlsx](https://github.com/SMSK-CO/SMSK/files/1519818/ExampleForGitHub.xlsx)

@craigmcampbell GitHub is not allowing me to assign this to you. Not sure if you are notified if I don't assign you so I am doing it this way.

**Comment by craigmcampbell** [comment]
I'll take a look at this and see what is going on. I need to look closer at the data points you mentioned, but I can tell you that this child was seen a total of four times (ranging from 9/16 - 9/17) and the referrals in those four visits have three different values.

**Comment by craigmcampbell** [comment]
@thomasnatalie I'm looking into this issue, but wanted to let you know that I change your security role to an admin. That should allow you to assign things to others.

**Comment by craigmcampbell** [comment]
I'm confused about where the data you attached came from. I just ran the report and I noticed that the column headers are named differently and in a different order than what you sent me. I also see the correct data in the report that I just ran.

What you sent me looks like it was manipulated somehow to change the format. Is it possible an error was made that messed up the data?

Here is a comparison of the two sets of data: 
[ReportComparison.xlsx](https://github.com/SMSK-CO/SMSK/files/1540333/ReportComparison.xlsx)
